eventsProject/singleEventPage.php

```diff
@@ -42,9 +72,14 @@
 		<div class='centeredPictureText' style='border: 5px solid #521f66;'>
 			<div class='titleWithVotes'>
 				<div class='voteColumn' style='font-size:20px'>
-					<p> <img class='iconAligned' style='background-color:white' src='/pics/arrow2.jpg' alt='arrow' height=40px></p>
-					$event[votes]
-					<p> <img class='iconAligned' style='background-color:white' src='/pics/line2.jpg' alt='line' height=40px></p>
+					<br>
+					<a href='#' onclick='upVoteNumber($myEventVotes, $eventID);'><img class='iconAligned' style='background-color:white' src='/pics/arrow2.jpg' alt='arrow' height=40px></a>
+					<br><br>
+					<div id='votes'>
+						$event[votes]
+					</div>
+					<br>
+					<a href='#' onclick='downVoteNumber($myEventVotes, $eventID);'><img class='iconAligned' style='background-color:white' src='/pics/line2.jpg' alt='line' height=40px></a>
```

Comment:
This file looks great! You've obviously got the core concepts totally figured out. Now it's just a matter of learning more and more interesting ways of applying those concepts.